asset_dashboard/static/js/components/BaseTable.js

```diff
@@ -101,11 +63,12 @@ const BaseTable = ({ rows = [],  getTrProps = props => props, rowClassNames, sel
       
       
       <div className="row container" aria-label="Pagination for all potential projects.">
-          <div className="d-flex col justify-content-start align-items-center ml-2">
-            <small className="text-muted">Pages {pageIndex + 1} of {pageOptions.length}</small>
+          <div className="d-flex col align-items-center ml-2">
+            <small className="text-muted">Pages {pageIndex + 1} of {pageOptions.length > 0 ? pageOptions.length : 1 }</small>
```

Comment:
fixed this bug that was bothering me. also improved the css for the pagination elements

Reply:
(actually didn't fix it in the CIP planner tables...hmmm)

Reply:
also improved the css for the pagination elements

Reply:
ah, i realized why the code changes weren't showing. i didn't change the paths in the django view and template, so i was served the cached react. ccea10e fixes this.